requests/sessions.py

```diff
@@ -735,8 +735,11 @@ def session():
     """
     Returns a :class:`Session` for context-management.
 
-    This method is kept for backwards compatibility. For new code,
-    initialize :class:`Session` directly.
+    .. deprecated:: 1.0.0
+
+        This method has been deprecated since version 1.0.0 and is only kept for
+        backwards compatibility. New could must use :class:`~requests.session.Session`
```

Comment:
New *code*

Reply:
This was fixed in the following commit https://github.com/requests/requests/pull/4685/commits/5789e91ec7265d662cd0eb077db2b5bb9363b520